### PR TITLE
Docs: Add clarification note about full range histogram in Loki

### PR DESCRIPTION
**Pull request by kavirajk**
**What this PR does / why we need it**:
This PR adds special note to Full Range Log histogram docs when using Loki.

Adding it because we see lots of community folks in Loki public slack were asking for ways to control the timeout of Log Volume Queries if it's expensive.

**Which issue(s) this PR fixes**:

Fixes #NA

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/explore/logs-integration.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Results of log queries are shown as histograms in the graph and individual logs
 
 If the data source supports a full range log volume histogram, the graph with log distribution for all entered log queries is shown automatically. This feature is currently supported by Elasticsearch and Loki data sources.
 
+**NOTE:** In Loki, this full range log volume histogram is rendered by normal metric query which can be expensive depending on time range queried. This expensive query may be particularly challenging for smaller Loki installations to process. To migitage this, we recommend using a proxy (e.g., nginx) in front of Loki to set a custom timeout (e.g. 10 seconds) for these queries. Log volume histogram queries can be identified by looking for queries with the HTTP header `X-Query-Tags` with value `Source=logvolhist`; these headers are added by Grafana when issuing the queries.
```
Comment (09jvilla) [inline docs/sources/explore/logs-integration.md:29]:
```suggestion
**NOTE:** In Loki, this full range log volume histogram is rendered by metric query which can be expensive depending on time range queried. This query may be particularly challenging for smaller Loki installations to process. To mitigate this, we recommend using a proxy like [nginx](https://www.nginx.com/) in front of Loki to set a custom timeout (e.g. 10 seconds) for these queries. Log volume histogram queries can be identified by looking for queries with the HTTP header `X-Query-Tags` with value `Source=logvolhist`; these headers are added by Grafana to all log volume histogram queries. 
```

[the remark this answers is not in the document]
Comment (09jvilla) [reply docs/sources/explore/logs-integration.md:29]: Just fixing some typos in my original suggestion. 

I'd love to also have a snippet of config to link users to for how you could do this in nginx but I'm ok leaving that out of scope for this PR. I think that snippet should probably live in the Loki docs, not in the Grafana docs.

[the remark this answers is not in the document]
Comment (09jvilla) [reply docs/sources/explore/logs-integration.md:29]: Just fixing some typos in my original suggestion. 

I'd love to also have a snippet of config to link users to for how you could do this in nginx but I'm ok leaving that out of scope for this PR. I think that snippet should probably live in the Loki docs, not in the Grafana docs.

**Review by 09jvilla — CHANGES_REQUESTED** [review]

File: docs/sources/explore/logs-integration.md (hunk 1 above)
Comment (09jvilla) [reply docs/sources/explore/logs-integration.md:29]: Just fixing some typos in my original suggestion.

I'd love to also have a snippet of config to link users to for how you could do this in nginx but I'm ok leaving that out of scope for this PR. I think that snippet should probably live in the Loki docs, not in the Grafana docs.

Comment (kavirajk) [reply docs/sources/explore/logs-integration.md:29]: I agree, nginix snippet is out of the scope of the PR. Plus, there are tons of different proxies. Nginix is just one example.

Yea thought about linking it from the Loki docs(also good idea), but currently don't know the right place to put it though. Open to suggestions.

**Review by 09jvilla — APPROVED** [review]
LGTM.

**Pull request merged**